cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/apply/ApplicationRequestsPlaner.java

```diff
@@ -64,62 +69,83 @@ private void addCreateAppRequest(ApplicationBean affectedObject) throws Creation
 
     /**
      * Creates the requests for CfObjectValueChanged
+     * 
      * @param objectValueChanged the CfObjectValueChanged to be visited
      */
     @Override
     public void visitObjectValueChanged(CfObjectValueChanged objectValueChanged) {
-        //TODO: later US
+        // TODO: later US
         return;
     }
 
     /**
      * Creates the requests for CfRemovedObject
+     * 
      * @param removedObject the CfRemovedObject to be visited
+     * @throws ApplyException           if an error during the apply logic occurs.
+     *                                  May contain another exception inside with
+     *                                  more details
+     * @throws IllegalArgumentException if the newObject is neither an
+     *                                  ApplicationBean or an
+     *                                  ApplicationManifestBean
      */
     @Override
     public void visitRemovedObject(CfRemovedObject removedObject) {
-        //TODO: later US
+              
```

Comment:
I think it would be a good idea to add a precondition check that asserts wether the request list is still empty. Just in case there is a bug.